data-analytics/iot-nirvana/client/src/main/java/com/google/cloud/demo/iot/nirvana/common/CityTemperature.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.demo.iot.nirvana.common;
+
+/**
+ * Class that describes information about temperature collected by devices positioned in different
+ * cities
+ */
+public class CityTemperature {
```

Comment:
Why not use the same class that you have for the pipeline/ piece instead of repeating all that code? Lets move the common stuff to maybe a common/ package so we are not repeating the code? I understand you will have to add "temperature" back to the "City" class. Unfortunately I had not review the client module at that time.

Reply:
Created a 'common' module containing the City, Message, TemperatureUtils and FormatExceptio classes and made it so that all modules ('app-engine', 'client', 'pipeline') use it. Commits: 905a6318327f8c053af956d808e7b70659af6179, 0af873748b8d172663e9571ce40803f74eaa39f2, 16171076705d196fc80007333f48beb8c7e1a472.